src/strife/m_menu.c

```diff
@@ -2336,6 +2336,7 @@ boolean M_Responder (event_t* ev)
             {
                 key = key_menu_forward;
                 mousewait = I_GetTime() + 15;
+                if (menuindialog) // [crispy] fix mouse fire delay
```

Comment:
If you want to avoid introducing indentation, which is perfectly fine, please use bracelets instead.

Reply:
Fixed by https://github.com/fabiangreffrath/crispy-doom/pull/1014/commits/1357f52057d38032cd44a2f60d32ee93a024541a.